app/views/additional_payments/claims/current_school.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for(:page_title, page_title(t("questions.current_school"), journey: current_journey_routing_name, show_error: current_claim.errors.any?)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% if @schools.blank? %>
+
+    <%= render "school_search",
+               question: t("additional_payments.questions.current_school_search"),
+               school_param: :current_school,
+               school_id_param: :current_school_id,
+               school_search_value: current_claim.eligibility.current_school_name,
+               exclude_closed: true
+    %>
+
+    <% else %>
+      <%= render "school_search_results",
+                 question: t("additional_payments.questions.current_school_results"),
+                 school_param: :current_school,
+                 school_id_param: :current_school_id,
+                 exclude_closed: true
+      %>
+    <% end %>
+  </div>
+</div>
```

Comment:
The old version had two local variables passed to the partial templates; `question` and `ecp_question`. The reason was that of it was an ECP claim the partial should show different text, and there were conditionals in those partials. This change just splits the template so that only `question` is passed to the partial with the correct translation for that journey, so I could remove the conditional logic from the partials.